### PR TITLE
Included 2021 and 2023 CarpentryConnect's

### PR DESCRIPTION
**Pull request by elletjies**

### CODE REVIEW

File: case-studies.markdown [hunk 1]
```diff
@@ -15,6 +15,49 @@ figure img { border: 1px solid #999999;}
...
+
+### CarpentryConnect New Zealand
+
+Dates: 9 February 2021
+
+Location: Wellington, New Zealand and Online
+
+Event website: [https://www.nesi.org.nz/news/2021/02/highlights-carpentry-connect-2021-bringing-together-new-zealands-digital-skills](https://www.nesi.org.nz/news/2021/02/highlights-carpentry-connect-2021-bringing-together-new-zealands-digital-skills)
+
+For any questions about specific elements around the planning of this event, [please contact Megan Guidry](mailto:megan.guidry@nesi.org.nz)
```
Comment (acrall) [inline case-studies.markdown:41]: Megan no longer works here, so I think we might want to remove all contact information from the page unless it is a generic email address.

File: case-studies.markdown [hunk 2]
```diff
@@ -15,6 +15,49 @@ figure img { border: 1px solid #999999;}
...
+
+- [Carpentries Connect South Africa 2021 opening and closing keynotes now available in isiXhosa and isiZulu](https://www.codeforsociety.org/eventfund/updates/carpentries-connect-south-africa-2021-opening-and-closing-keynotes-now-available-in-isixhosa-and-isizulu) by Angela Okune
+- [CarpentryConnect South Africa 2021 in Review](https://eresearch.uwc.ac.za/2021/10/06/carpentry-connect-south-africa-2021-in-review/) by the University of the Western Cape
+- [CarpentryConnect South Africa 2021 - A global event](https://carpentries.org/blog/2021/09/carpentryconnect/) by Angelique Trusler
+- [Highlighting our strategic plan efforts to support local community building](https://carpentries.org/blog/2021/07/Highlighting-our-strategic-plan-efforts-to-support-local-community-building/) by Kari L. Jordan and Angelique Trusler
+- [Registration for CarpentryConnect South Africa 2021 now open!](https://carpentries.org/blog/2021/06/CarpentryConnectSouthAfrica/) by Angelique Trusler
+
+For any questions about specific elements around the planning of this event, [please contact Angelique Trusler](mailto:admin-afr@carpentries.org), African Capacity Development Manager.
```
Comment (acrall) [inline case-studies.markdown:59]: Here, perhaps you could remove your name and leave the title since the email is not your personal one?

**Review by acrall — APPROVED** [review]
Just a couple of minor edits associated with contact information

**Comment by elletjies** [comment]
@acrall thanks for the suggestions, they were incorporated and I merged the PR

**Pull request merged**